eventdisplay/tracks/FairEveGeoTracks.cxx

```diff
@@ -49,8 +49,10 @@ InitStatus FairEveGeoTracks::Init()
 {
     FairRootManager *mngr = FairRootManager::Instance();
     fContainer = (TClonesArray *)mngr->GetObject("GeoTracks");
-    if (fContainer == nullptr)
-        return kFATAL;
+    if (!fContainer) {
+        LOG(warning) << "Geo tracks not found !";
```

Comment:
Could you please add a short explanation what will happen due to the absence of the Geo Tracks. Probably something like "deactivating task".

Please due the same also for the other error messages. It would help the users to understand the consequence.